scripts/createImage.sh

```diff
@@ -5,8 +5,8 @@ set -e
 
 source scripts/init.sh
 
-cp versions/${SPARK_VERSION}/pom.xml pom.xml
-cp versions/${SPARK_VERSION}/SparkSubmit.scala src/main/scala/org/apache/spark/deploy/ArmadaSparkSubmit.scala
+# Set this project's Spark and Scala version according to the Spark sources in $SPARK_HOME
+./scripts/set-version "$SPARK_VERSION" $"SCALA_VERSION"
```

Comment:
After running this a few times, I discovered this line should actually be:

```suggestion
./scripts/set-version.sh $SPARK_VERSION $SCALA_VERSION
```